lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/driver/VortexMasterConf.java

```diff
@@ -97,6 +104,11 @@
   public static final RequiredParameter<Integer> NUM_OF_VORTEX_START_THREAD = new RequiredParameter<>();
 
   /**
+   * Size of threadpool for callbacks on VortexFuture.
+   */
+  public static final OptionalParameter<Integer> FUTURE_CALLBACK_THREADPOOL_SIZE = new OptionalParameter<>();
```

Comment:
Here also, as we use `ThreadPool`, I think this would be `FUTURE_CALLBACK_THREAD_POOL_SIZE`